compiler/src/dotty/tools/dotc/report.scala

```diff
@@ -151,7 +151,7 @@ object report:
 
     val info1 = formatExplain(List(
       "while compiling"    -> ctx.compilationUnit,
-      "during phase"       -> ctx.phase.prevMega,
+      "during phase"       -> ctx.phase.megaPhase,
```

Comment:
Why not `ctx.phase` ?

Reply:
Because otherwise we get the first phase of the megaphase, which is misleading.  The megaphase is what -Xprint show's, which is my reference point.